code/upaya/config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,36 @@
+SecureHeaders::Configuration.default do |config|
+  config.hsts = "max-age=#{1.day.to_i}; includeSubDomains"
+  config.x_frame_options = 'SAMEORIGIN'
+  config.x_content_type_options = 'nosniff'
+  config.x_xss_protection = '1; mode=block'
+  config.x_download_options = 'noopen'
+  config.x_permitted_cross_domain_policies = 'none'
+  config.csp = {
+    default_src: %w('self'),
+    report_only: Rails.env.development? ? true : false,
+    frame_src: %w('self'), # deprecated in CSP 2.0
+    child_src: %w('self'), # CSP 2.0 only; replaces frame_src
+    # frame_ancestors: %w('self'), # CSP 2.0 only; overriden by x_frame_options in some browsers
+    preserve_schemes: true, # needed to temporarily allow mixed content
```

Comment:
This can be removed. It defaults to `false`, which is what we want. We only set it to `true` in ICAM to allow testing with the demo-live SP.